Sources/SkyFloatingLabelTextFieldWithIcon.swift

```diff
@@ -218,7 +236,22 @@ open class SkyFloatingLabelTextFieldWithIcon: SkyFloatingLabelTextField {
     /// Update the colors for the control. Override to customize colors.
     override open func updateColors() {
         super.updateColors()
-        updateIconLabelColor()
+        if self.iconType == .font {
+            updateIconLabelColor()
+        }
+        if self.iconType == .image {
+            updateImageViewTintColor()
+        }
+    }
+
+    fileprivate func updateImageViewTintColor() {
+        if !isEnabled {
```

Comment:
Shouldn't this only be done if `templateImage` is `true`?

Reply:
Change added. Although not doing so has no side effects